src/viewer.js

```diff
@@ -2132,6 +2117,55 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
      */
     _cancelPendingImages: function() {
         this._loadQueue = [];
+    },
+
+    /**
+     * Remove the reference strip.
+     * @param {Boolean} hide - Disables automatic showing of the reference strip in the future until manually re-enabled.
+     * @function
+     */
+    removeReferenceStrip: function(hide) {
+        if (hide) {
+            this.showReferenceStrip = false;
+        }
+
+        if (this.referenceStrip) {
+            this.referenceStrip.destroy();
+            this.referenceStrip = null;
+        }
+    },
+
+    /**
+     * Display the reference strip based on the currently set tileSources.
+     * @function
+     * @return {OpenSeadragon.Viewer} Chainable.
```

Comment:
Please remove the chaining comment and the `return this` lines from this function.